ec2/sc-product-ec2-windows-jumpcloud.yaml

```diff
@@ -43,16 +38,18 @@ Resources:
   Associateec2windows:
     Type: AWS::ServiceCatalog::PortfolioProductAssociation
     Properties:
-      PortfolioId: !Ref 'PortfolioId'
+      PortfolioId: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-portfolio-ec2-SCEC2portfolioId'
       ProductId: !Ref 'scec2windowsproduct'
   constraintec2linux:
     Type: AWS::ServiceCatalog::LaunchRoleConstraint
     DependsOn: Associateec2windows
     Properties:
-      PortfolioId: !Ref 'PortfolioId'
+      PortfolioId: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-portfolio-ec2-SCEC2portfolioId'
       ProductId: !Ref 'scec2windowsproduct'
-      RoleArn: !Ref 'LaunchConstraintARN'
-      Description: !Ref 'LaunchConstraintARN'
+      RoleArn: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-ec2vpc-launchrole-LaunchRoleArn'
 Outputs:
```

Comment:
I think you need to export the `ProductId`.

Reply:
good catch.  i've been neglecting windows.